### ISSUE TITLE
Improve 'Creating Additional Docs from App Forms'

### ISSUE THREAD

**Issue by binokaryg** [medic/cht-docs#1126]
Page: [/apps/guides/forms/additional-docs/](https://docs.communityhealthtoolkit.org/apps/guides/forms/additional-docs/)
Version: 2022-06-17 14:30:45 -0500 -0500
---
![image](https://github.com/medic/cht-docs/assets/20041732/9b982df1-6f48-486f-84c8-8609920b48a4)

The description: `Integration for sending and receiving SMS` is not right. It's probably leftover from [this page](https://docs.communityhealthtoolkit.org/apps/guides/messaging/gateways/africas-talking/).

**Comment by Sub-codes** [comment]
Hey is this issue still open

**Comment by binokaryg** [comment]
> Hey is this issue still open

Yes, the sub-header is still there in the page: [/apps/guides/forms/additional-docs/](https://docs.communityhealthtoolkit.org/apps/guides/forms/additional-docs/)

**Comment by binokaryg** [comment]
Do you want to get it assigned to you?

**Comment by Sub-codes** [comment]
Yes I am new to open source and want to give it a try and can you clarify what changes do I need to make?

**Comment by binokaryg** [comment]
I think you can just remove the highlighted text: `Integration for sending and receiving SMS`

**Comment by Sub-codes** [comment]
So can you assign this to me i will give it a try

**Comment by binokaryg** [comment]
> So can you assign this to me i will give it a try

Assigned.

**Comment by mrjones-plip** [comment]
Fixed!  Thanks @Sub-codes 

![Image](https://github.com/user-attachments/assets/80446027-d051-4902-828e-d08b35127067)

**Comment by Sub-codes** [comment]
Hoping to contribute 😊

**Comment by andrablaj** [comment]
Great catch @Sub-codes; the other issue was closed. Thank you!
